Sources/ApolloCodegenLib/Templates/MockObjectTemplate.swift

```diff
@@ -21,16 +21,19 @@ struct MockObjectTemplate: TemplateRenderer {
 
   var template: TemplateString {
     let objectName = graphqlObject.formattedName
-    let fields: [TemplateField] = ir.fieldCollector
+    let fields: [(TemplateField, GraphQLType)] = ir.fieldCollector
```

Comment:
Turned this into an array of tuples to keep the `GraphQLType` together with the `TemplateField` to check the type to determine the appropriate `_set` function to call.

Reply:
Since `TemplateField` is already a tuple should we just change the `type` field of that tuple to the `GraphQLType` and do the formatting of the `rendered(as: .testMockField)` in the rendering on like 52?